### PR TITLE
Fix grpc falco errors

### PR DESCRIPTION
**Pull request by sleipnir**
Hi @LesnyRumcajs 

As promised, follow fix of https://github.com/LesnyRumcajs/grpc_bench/pull/198
Let me know if it helps.
Thanks

### CODE REVIEW

**Comment by sleipnir** [comment]
@LesnyRumcajs  I ended up committing some dependencies, please forgive me, but this shouldn't have any impact on the build

**Review by LesnyRumcajs — COMMENTED** [review]
@sleipnir Thank you. Are you sure those additional dependencies are necessary? Seems like build artifacts.

**Comment by sleipnir** [comment]
> @sleipnir Thank you. Are you sure those additional dependencies are necessary? Seems like build artifacts.

No. I send by mistake

File: elixir_grpc_bench/lib/helloworld.pb.ex [hunk 1]
```diff
@@ -0,0 +1,91 @@
+defmodule Helloworld.Hello.Pet.Color do
```
Comment (LesnyRumcajs) [inline elixir_grpc_bench/lib/helloworld.pb.ex:1]: can this file be also generated during build phase and not added to version control?

Comment (LesnyRumcajs) [reply elixir_grpc_bench/lib/helloworld.pb.ex:1]: @sleipnir ?

Comment (sleipnir) [reply elixir_grpc_bench/lib/helloworld.pb.ex:1]: Hi @LesnyRumcajs, I've been away for a few days, I'll take a look at this as soon as possible.

**Pull request merged**